ipatests/test_integration/test_membermanager.py

```diff
@@ -212,3 +212,54 @@ def test_hostgroup_member_manager_nopermission(self):
             "'write' privilege to the 'memberManager' attribute of entry"
         )
         assert expected in result.stdout_text
+
+    def test_member_manager_upgrade_scenario(self):
+        """tmp"""
+        user = "idmuser"
+        password = "Secret123"
+        group1 = "role-groupmanager"
+        group2 = "role-usergroup-A"
+
+        master = self.master
+
+        tasks.kinit_admin(master)
+        tasks.group_add(master, group1)
+        tasks.group_add(master, group2)
+        tasks.create_active_user(master, user, password)
+
+        tasks.kinit_admin(master)
+        master.run_command(["ipa", "group-add-member", group1, "--user", user])
+        master.run_command(["ipa", "group-add-member-manager", "--groups",
+                            group1, group2])
+
+        replica = self.replicas[0]
```

Comment:
In this test there is no replica: the class defines `topology = "line"` but does not override the `num_replicas = 0` attribute set by default in the class IntegrationTest.

The test doesn't need a replica, you can use the information from self.master instead.